app/containers/Forms/SummaryFormFieldTemplate.tsx

```diff
@@ -9,10 +9,17 @@ const FormFieldTemplate: React.FunctionComponent<FieldTemplateProps> = ({
   children,
   schema,
   classNames,
-  hidden
+  hidden,
+  uiSchema
 }) => {
   if (hidden) return null;
 
+  if (
+    uiSchema['ui:options']?.disableRenderingIfEmpty &&
+    !children[0]?.props?.formData
+  )
+    return null;
+
```

Comment:
move this below the two `if` statements, to ensure that the schema is neither an object nor an array. 
To ensure that you render the field if it has a value of `0`, you should use
```suggestion
  if (
    uiSchema['ui:options']?.disableRenderingIfEmpty &&
    (children[0]?.props?.formData === null || children[0]?.props?.formData === undefined)
  )
    return null;

```

Reply:
Good catch! although if the comment has a value of numeric 0, something has gone wrong since it should be a string with value `"0"`. That might be confusing for the user to see "comments: 0" when they haven't entered anything?

Reply:
My reasoning here would be for other fields that would eventually use that uiSchema option in the future, not necessarily for the comments form

Reply:
Excellent point 👍